en/lessons/gravity-model.md

```diff
@@ -0,0 +1,712 @@
+
+| Variable             | Weighting  | Symbol   |
+| -------------        |:-------------:      | :------: |
+| y-intercept          | -3.84678            | β0        |
+| population           | 1.235208               | β1		|
+| distance             | -0.541863              | β2		|
+| wheat price          | -0.023957               | β3		|
+| wages                | -0.025184              | β4		|
+| wage trajectory      | -0.013779              | β5      |
+
+To calculate these values long-hand requires an incredible amount of work. We will use a quick solution in the *R* programming language that takes advantage of William Venables and Brian Ripley's *MASS* package that can solve negative binomial regression equations like our gravity model with a single line of code. However, it is important to understand the principles behind what one is doing in order to appreciate what the code does (note the following sections do not DO the calculation, but explain its steps for you; we will do the calculation with the code further down the page).
+
+**Calculating the individual Weightings (in principle)**
+
+$$β_{1}$$, $$β_{2}$$, etc, are the same as "$$β$$" in the Simple Linear Regression model above, which is the [slope](https://en.wikipedia.org/wiki/Slope) of the regression line (the rise over the run, or how much "$$y$$" increases when "$$x$$" increases by 1). The only difference here between a Simple Linear Regression and our gravity model is that we have to calculate 5 slopes instead of 1.
+
+*A Simple Linear Regression*
+$$y = α + βx$$
+
+We will need to solve for each of these five slopes before we can calculate the y-intercept in the next step. That is because the slopes of the various $$β$$ values are part of the equation for calculating the y-intercept.
+
+The formula for calculating β in a regression analysis is:
+
+$$β = r (\frac{s_{y}}{s_{x}})$$
+
+- We already know that "$$β$$" is the slope, which is what we are trying to calculate.
+- "r" is [Pearson's correlation coefficient](https://en.wikipedia.org/wiki/Pearson_correlation_coefficient), which we are going to compute
+- "$$s_{y}$$" is the standard deviation of "$$y$$"
+- "$$s_{x}$$" is the standard deviation of "$$x$$"
+
+**Pearson's Correlation Coefficient**
+
+Pearson's correlation coefficient can be calculated long-hand but it's a rather long calculation in this case, requiring 64 numbers. There are some great video tutorials in English available online if you would like to see a walk-through of how to do the calculations long-hand.[^17] There are also a number of online calculators that will calculate "r" for you if you provide the data. Given the large number of digits to compute, I would recommend a website with a built in tool designed to make this calculation. Make sure you choose a reputable site, such as one offered by a university.
```

Comment:
As above, `"r"` should be listed as `$$r$$` for consistency

Reply:
done.